sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -62,6 +62,15 @@ public FormTrainingClient(Uri endpoint, AzureKeyCredential credential, FormRecog
             ServiceClient = new ServiceClient(diagnostics, pipeline, endpoint.ToString());
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FormTrainingClient"/> class.
+        /// </summary>
+        /// <param name="serviceClient">Provides communication with the Form Recognizer Azure Cognitive Service through its REST API.</param>
+        internal FormTrainingClient(ServiceClient serviceClient)
+        {
+            ServiceClient = serviceClient;
```

Comment:
Is there no issue with holding a reference to the same ServiceClient from multiple FR clients?

Reply:
Actually we could hold the same ServiceClient in multiple FormTrainingClients, but all from a single FormRecognizerClient.

Since they can't be disposed and are thread-safe, I can't think of issues we could have because of that. Asked Pavel and there are no significant perf differences we could end up with.